wallet/src/Pos/Util/BackupPhrase.hs

```diff
@@ -28,21 +29,15 @@ newtype BackupPhrase = BackupPhrase
     { bpToList :: [Text]
     } deriving (Eq, Generic)
 
+-- NOTE: it's guaranteed not to fail
 instance Arbitrary BackupPhrase where
-    arbitrary = BackupPhrase <$> vectorOf 12 (elements englishWords)
+    arbitrary = either error identity <$> arbitraryMnemonic 16
```

Comment:
Instead of failing at runtime, we could instead recur until we get a valid value:

```haskell
arbitrary = do
    em <- arbitraryMnemonic 16
    case em of
          Left _ -> arbitrary
          Right a -> pure a
```